plugins/doc_fragments/template.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2022, 2023
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+
+    DOCUMENTATION = r'''
+options:
+  use_template:
+    description:
+      - Whether the module should treat C(src) as a Jinja2 template and
```

Comment:
Do you know which version of Jinja2 is supported? do we select it or are dependent on python version?

Reply:
I don't know for sure which version of Jinja is installed by each version of Ansible, but since it's a dependency for the engine, we don't really control it.